exercises/04-integration/after/Divergent.Customers/ServiceHelperClassic.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Divergent.Customers
+{
+    static class ServiceHelper
+    {
+        public static bool IsService()
+        {
+            return !Environment.UserInteractive;
+        }
+    }
+}
```

Comment:
This is used in one place only. I'd prefer to get rid of it and have the caller call `Environment.UserInteractive` instead.

Reply:
@bording did a [PR to clean this template](https://github.com/Particular/dotnetTemplates/pull/3)